incident_io/README.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+
+[incident.io][1] helps companies declare, collaborate, communicate around, and learn from events that disturb their normal course of business-from critical infrastructure being down, to data breaches and security incidents. It is a service that helps teams manage incidents and outages effectively. It typically provides features like incident reporting, tracking, and resolution workflows.
+
+Integrate your incident.io account with Datadog to gain insights into incident-related activities.
+
+## Setup
+
+Follow the instructions below to configure this integration for incident.io incident events through a Webhook.
+
+### Configuration
+
+#### Webhook configuration
+Configure the Datadog endpoint to forward events of incident.io incidents as logs to Datadog. For more details, see the incident.io [webhooks][2] documentation.
+
+1. Select an existing API key or create a new one by clicking one of the buttons below: <!-- UI Component to be added by Datadog team -->
```

Comment:
Not sure what the backstory is to this placeholder, but do we just want users to generate a Datadog API key here? If so, we can link them to `https://app.datadoghq.com/organization-settings/api-keys`.

Reply:
I've actually taken care of this in this task.  I updated the configuration tab to match our other webhooks integrations.  https://datadoghq.atlassian.net/browse/SAASINT-3196